mowgli_etl/pipeline/wdc/Data_Exploration/offers_corpus_english_v2_clean.py

```diff
@@ -0,0 +1,47 @@
+from os import listdir
+
+# Generate a list of all possible jsonl files
+directory = [item for item in listdir() if ".jsonl" in item]
+
+# Present options for jsonl files
+print("Please choose which of the following files you'd like to clean [enter number on the left]:\n")
+for i in range(0, len(directory)):
+	print("\t{} - {}".format(i, directory[i]))
+
+# Have user select which file to clean
+choice = directory[int(input("\nChosen file number: "))]
+
+# Copy each line in the file
+lines = [line for line in open(choice, "r")]
+
+# Create cleaned file
+with open(choice[0:-6]+"_clean.jsonl", "w") as new_file:
+	danger_string = "\"brand\":"
+	for line in lines:
+		# Catch lines with more than one object based on brand identifier
+		if line.count(danger_string) == 1:
+			new_file.write(line)
```

Comment:
In situations like this, where you have a short branch of an if-else nested in a loop, the usual idiom is to do

```
if condition:
   ... a few statements ...
   continue
... else branch code here ...
```

or equivalent with the else branch. That saves having to indent the else.

You can do the same thing with return from a function, which is one of the advantages of having many small functions.